lib/handlers/sdp/RemoteSdp.js

```diff
@@ -267,6 +271,13 @@ class RemoteSdp {
             this._sdpObject.media[idx] = newMediaSection.getObject();
         }
     }
+    _findMediaSection(mid) {
```

Comment:
I don't understand why we may need this method, we are supposed to already be able to identify the media section somehow else in many other methods. Please remember that we are ready for Unified and Plan B in RemoteSdp class.

Reply:
I created this private helper method only to avoid the code duplication.   This code to find a mediaSection from a mid was duplicated in `closeMediaSection()` and `disableMediaSection()`.